CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+## When to submit an ADR
+
+The purpose of an ADR is to record an over-arching technical decision made by the development team in the Archivematica 
```

Comment:
Is it worth describing our roles? `The development team consists of software developers, archivists, and digital preservation specialists.'?

Reply:
Good idea, I added that change!